### PR TITLE
[CircleCI] Avoid auto updating Homebrew on installing `swiftlint`

### PR DESCRIPTION
**Pull request by norio-nomura**
This reduces the `swiftlint lint` step from 35-120 seconds to few seconds.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ aliases:
     - checkout
     - run: git submodule update --init --recursive
     - run: swift test
-    - run: brew install swiftlint && swiftlint lint --strict
+    - run: brew install https://raw.github.com/Homebrew/homebrew-core/master/Formula/swiftlint.rb && swiftlint lint --strict
```
Comment (keith) [inline .circleci/config.yml]: If you'd prefer you can instead set `HOMEBREW_NO_AUTO_UPDATE=1`

Comment (norio-nomura) [reply .circleci/config.yml]: That would make it possible to use the old `swiftlint`.

Comment (keith) [reply .circleci/config.yml]: Ah yea, you might want to lock this anyways so an update to an existing rule doesn't break PRs unexpectedly

Comment (norio-nomura) [reply .circleci/config.yml]: Ah, I needed `HOMEBREW_NO_AUTO_UPDATE = 1` for what I intended. Thank you for teaching me.
Duration of `swiftlint` step:
2:00 before: https://circleci.com/gh/jpsim/SourceKitten/3682
6sec after: https://circleci.com/gh/jpsim/SourceKitten/3690

Comment (norio-nomura) [reply .circleci/config.yml]: > Ah yea, you might want to lock this anyways so an update to an existing rule doesn't break PRs unexpectedly

I think that we are expecting to use the latest swiftlint.

**Comment by jpsim** [comment]
Thanks!

**Pull request merged**